index.bs

```diff
@@ -666,16 +677,14 @@ never exceed the [=sampling frequency=] for the given [=sensor type=].
 ## Conditions to expose sensor readings ## {#concepts-can-expose-sensor-readings}
 
 The user agent must verify that all [=mandatory conditions=] are satisfied to ensure it
-<dfn>can expose sensor readings</dfn>.
+<dfn>can expose sensor readings</dfn> to an [=active document=].
```

Comment:
maybe change "to a given [=active document=]."

Reply:
done